.gitignore

```diff
@@ -79,3 +79,5 @@ yarn-debug.log*
 /.envrc
 /.nix
 /.ruby-lsp
+
+.tool-versions
```

Comment:
So ideally we should all be on the same versions of stuff. If you wanted to commit this file that's actually quite helpful to know that we're all running/installing the same versions of languages/tools